OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -787,6 +790,27 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
             return null;
         }
 
+
+        public static ProcStatus? BitShiftLeftReference(DMProcState state) {
+            DMReference reference = state.ReadReference();
+            DreamValue second = state.Pop();
+            DreamValue first = state.GetReferenceValue(reference, peek: true);
+            DreamValue result;
+            switch (first.Type) {
+                case DreamValue.DreamValueType.DreamObject when first == DreamValue.Null:
+                    result = new DreamValue(0);
+                    break;
+                case DreamValue.DreamValueType.Float when second.Type == DreamValue.DreamValueType.Float:
+                    result = new DreamValue(first.GetValueAsInteger() << second.GetValueAsInteger());
```

Comment:
`GetValueAsInteger()` is deprecated. This should be converted to use the `TryGetValueAs[]()` variants instead of checking the type in a `switch`.